lib/ASTGen/Sources/ASTGen/SourceFile.swift

```diff
@@ -20,15 +25,31 @@ struct ExportedSourceFile {
   let syntax: SourceFileSyntax
 }
 
+extension Parser.ExperimentalFeatures {
+  init(from context: BridgedASTContext?) {
+    self = []
+    guard let context = context else { return }
+
+    func mapFeature(_ bridged: BridgedFeature, to feature: Self) {
+      if ASTContext_langOptsHasFeature(context, bridged) {
+        insert(feature)
+      }
+    }
```

Comment:
I assume this is a stub to be used once you introduce features that need to be mapped to SwiftParser, correct?

Reply:
Yeah